msg/CMakeLists.txt

```diff
@@ -151,6 +154,30 @@ set(msg_files
 	wind_estimate.msg
 	)
 
+set(deprecated_msgs
+	# TODO: uncomment as soon as https://github.com/PX4/Firmware/pull/13127 is rebased over this
+	# ekf2_innovations.msg	# 2019-11-22, Updated estimator interface and logging; replaced by 'estimator_innovations'.
+	)
+
+foreach(msg IN LISTS deprecated_msgs)
+	if(msg IN_LIST msg_files)
+		get_filename_component(msg_we ${msg} NAME_WE)
+		list(APPEND invalid_msgs ${msg_we})
+	endif()
+endforeach()
+if(invalid_msgs)
+	list(LENGTH invalid_msgs invalid_msgs_size)
+	if(${invalid_msgs_size} GREATER 1)
+		foreach(msg IN LISTS invalid_msgs)
+			string(CONCAT invalid_msgs_cs ${invalid_msgs_cs} "'${msg}', ")
```

Comment:
Why not use [join](https://cmake.org/cmake/help/latest/command/list.html#join)?

Reply:
`JOIN` only available in CMake 3.12 and above, and we are setting the min version to 3.2

Reply:
We could probably raise the minimum cmake version to 3.5 (what's packaged in ubuntu 16.04) at this point.